data/campaigns/Heir_To_The_Throne/utils/httt_utils.cfg

```diff
@@ -347,8 +350,9 @@ fire:  +10%"
         first_time_only=no
         [filter]
             side=1
-            x=22
-            y=32
+            [filter_location]
+                location_id="sceptre"
+            [/filter_location]
```

Comment:
Huh? It was originally hard-coded as 22,32? How did that even work? More importantly, what the heck are those two parameters at the top of the event? Are they relics of the old generator that can be removed?